docs/manual/programmability/lua-api.md

```diff
@@ -187,23 +187,28 @@ The helper accepts a single string argument and returns a Lua table with the _er
 The outcome of the following code is that _error1_ and _error2_ are identical for all intents and purposes:
 
 ```lua
-local text = 'My very special error'
+local text = 'ERR My very special error'
 local reply1 = { err = text }
 local reply2 = redis.error_reply(text)
 ```
 
 Therefore, both forms are valid as means for returning an error reply from scripts:
 
 ```
-redis> EVAL "return { err = 'My very special table error' }" 0
-(error) My very special table error
-redis> EVAL "return redis.error_reply('My very special reply error')" 0
-(error) My very special reply error
+redis> EVAL "return { err = 'ERR My very special table error' }" 0
+(error) ERR My very special table error
+redis> EVAL "return redis.error_reply('ERR My very special reply error')" 0
+(error) ERR My very special reply error
 ```
 
 For returning Redis status replies refer to [`redis.status_reply()`](#redis.status_reply).
 Refer to the [Data type conversion](#data-type-conversion) for returning other response types.
 
+**Note:**
+By convention, Redis uses the first word of an error string as a unique error code for
+specific errors or `ERR` for general-purpose errors. Scripts are advised to follow this
+convention, as shown in the example above, but this is not mandatory.
+
```

Comment:
Note that we don't respect the people 80-char line limit anymore.
Instead, we break after a period.

```suggestion
**Note:**
By convention, Redis uses the first word of an error string as a unique error code for specific errors or `ERR` for general-purpose errors.
Scripts are advised to follow this convention, as shown in the example above, but this is not mandatory.

```